data/projects/semesters/f21-w22/mealcare-revamp.js

```diff
@@ -0,0 +1,62 @@
+const name = 'MealCare Revamp';
+const id = 'mealcare-revamp';
+const detail = "A revamped mobile app to help volunteers track food waste and a data backend for in-depth data analysis.";
+const coverImagePath = "/static/images/projects/mealcare/mealcare_problem_blue.svg";
+const clientDetail = 'MealCare is a student-founded non-profit dedicated to reducing food waste and food insecurity across Canadian communities.';
```

Comment:
MealCare is a student-founded non-profit dedicated to reducing food waste and food insecurity across Canadian communities. This is done through volunteers who transport excess food from restaurants to homeless shelters and soup kitchens.